test/fixtures/all_passing_with_debugging/results.json

```diff
@@ -0,0 +1,18 @@
+{
+    "status": "pass",
+    "message": null,
+    "tests": [
+        {
+            "name": "first test",
+            "status": "pass",
+            "message": null,
+            "output": "x = 1\nI'M HERE!\n"
```

Comment:
Am I correct in seeing that the all the test output is captured and added to each test? I assume this is due to some limitation in getting output linked to a specific test, right?

Reply:
Yes, but it's also how the output would be displayed locally in Julia.

Reply:
Okay. Then it makes perfect sense.